### DISCUSSION TITLE
About Options.spec.ts

### DISCUSSION THREAD

**Discussion by RyoMasumura1201** [diballesteros/nuzlocke#91] [Q&A] [answered]
@diballesteros 
I have a question about cypress Options.spec.ts.
In the Report, why is there two spaces between Emerald and Kaizo, but only one space in have.value?
```
cy.get('[data-testid=report-selectedgame]')
      .type('Emerald  Kaizo')
      .should('have.value', 'Emerald Kaizo');
```
Why does this test succeed in the master branch?
(When I run this test on my local branch, the test fails as shown in the screenshot.)

I look forward to hearing from you soon.
<img width="1202" alt="スクリーンショット 2021-08-29 11 46 31" src="https://user-images.githubusercontent.com/65020585/131236655-b3dfaaf2-7945-4ffe-b067-267969f72cdb.png">

**Comment by diballesteros** [comment, marked as answer]
Good catch! It's a bug I forgot to create an issue for although with the sidebar it should't happen anymore. Basically since the entire modal and its component is **inside** of the menu a spacebar would cause the same menu to open/close in the background (behind the modal) and not register the space ' ' itself. So in the **.type** I had to enter the space twice.

I'm not sure of the behavior once its a sidebar but I expect it can be left as .type('Emerald Kaizo').

If you check out the following line:
 .type('Please  implement  new  feature')
 
 It also has double spaces. This can also be left with single spaces in the .type()

**Reply by RyoMasumura1201** [reply]
@diballesteros
Thanks for your answer!
I understand, I'll fix the test of report.
